core/src/main/java/org/dozer/builder/model/jaxb/ClassDefinition.java

```diff
@@ -0,0 +1,269 @@
+        return isAccessible;
+    }
+
+    protected void setClazz(String clazz) {
+        this.clazz = clazz;
+    }
+
+    protected void setBeanFactory(String beanFactory) {
+        this.beanFactory = beanFactory;
+    }
+
+    private void setFactoryBeanId(String factoryBeanId) {
+        this.factoryBeanId = factoryBeanId;
+    }
+
+    private void setMapSetMethod(String mapSetMethod) {
+        this.mapSetMethod = mapSetMethod;
+    }
+
+    private void setMapGetMethod(String mapGetMethod) {
+        this.mapGetMethod = mapGetMethod;
+    }
+
+    private void setCreateMethod(String createMethod) {
+        this.createMethod = createMethod;
+    }
+
+    private void setMapNull(Boolean mapNull) {
+        this.mapNull = mapNull;
+    }
+
+    private void setMapEmptyString(Boolean mapEmptyString) {
+        this.mapEmptyString = mapEmptyString;
+    }
+
+    private void setAccessible(Boolean accessible) {
+        isAccessible = accessible;
+    }
+
+    // Fluent API
+    //-------------------------------------------------------------------------
+    public ClassDefinition withClazz(String clazz) {
+        setClazz(clazz);
+
+        return this;
+    }
+
+    public ClassDefinition withBeanFactory(String beanFactory) {
+        setBeanFactory(beanFactory);
+
+        return this;
+    }
+
+    public ClassDefinition withFactoryBeanId(String factoryBeanId) {
+        setFactoryBeanId(factoryBeanId);
+
+        return this;
+    }
+
+    public ClassDefinition withMapSetMethod(String mapSetMethod) {
+        setMapSetMethod(mapSetMethod);
+
+        return this;
+    }
+
+    public ClassDefinition withMapGetMethod(String mapGetMethod) {
+        setMapGetMethod(mapGetMethod);
+
+        return this;
+    }
+
+    public ClassDefinition withCreateMethod(String createMethod) {
+        setCreateMethod(createMethod);
+
+        return this;
+    }
+
+    public ClassDefinition withMapNull(Boolean mapNull) {
+        setMapNull(mapNull);
+
+        return this;
+    }
+
+    public ClassDefinition withMapEmptyString(Boolean mapEmptyString) {
+        setMapEmptyString(mapEmptyString);
+
+        return this;
+    }
+
+    public ClassDefinition withAccessible(Boolean accessible) {
+        setAccessible(accessible);
+
+        return this;
+    }
+
+    public MappingDefinition end() {
+        return parentMappingDefinition;
+    }
+
+    public ConverterTypeDefinition endType() {
+        return parentConverterTypeDefinition;
+    }
+
+    public DozerClass build(BeanContainer beanContainer) {
+        DozerClass dozerClass = new DozerClass(beanContainer);
+        dozerClass.setName(clazz);
+        dozerClass.setBeanFactory(beanFactory);
+        dozerClass.setFactoryBeanId(factoryBeanId);
+        dozerClass.setMapGetMethod(mapGetMethod);
+        dozerClass.setMapSetMethod(mapSetMethod);
+        dozerClass.setCreateMethod(createMethod);
+        dozerClass.setMapNull(mapNull == null ? null : mapNull);
```

Comment:
Maybe we could just pass the `mapNull` as the result of ternary operator will be always equal to the `mapNull`?